src/service/plugins/mpris.js

```diff
@@ -228,6 +228,12 @@ var Plugin = GObject.registerClass({
             }
 
             // Player Properties
+            if (packet.body.hasOwnProperty('setLoopStatus'))
+                player.LoopStatus = packet.body.setLoopStatus;
+
+            if (packet.body.hasOwnProperty('setShuffle'))
+                player.Shuffle = packet.body.setShuffle === 'true';
```

Comment:
I'm not sure I see the reason for coercion and comparison here. I think kdeconnect-android should just send a real boolean and that should be passed to the proxy.

Reply:
https://invent.kde.org/network/kdeconnect-android/-/merge_requests/255, will force push this away when that gets merged. Thanks!